README.md

```diff
@@ -276,16 +292,15 @@ How to release a new triton-origin image:
    a few extra things are required:
 
    - Add entries to the "compatibility" tables in the "sdcnode compatibility
-     with triton-origin images" and "Joyent Jenkins agent labels compatibility
+     with triton-origin images" and "Jenkins agent labels compatibility
```

Comment:
This doesn't have an end quote after `"Jenkins`

Reply:
It's there, it's just on the next line which was unmodified.

Reply:
Oh, I see it. Sorry. I swear I had looked for it before.